src/app/containers/Figure/index.jsx

```diff
@@ -1,19 +1,36 @@
-import React, { useContext, Fragment } from 'react';
+import React, { useContext, Fragment, useState } from 'react';
 import { string, number, objectOf, any, bool } from 'prop-types';
 import Figure from '@bbc/psammead-figure';
 import Image, { AmpImg } from '@bbc/psammead-image';
 import ImagePlaceholder from '@bbc/psammead-image-placeholder';
 import LazyLoad from 'react-lazyload';
+import { Transition } from 'react-transition-group';
+import styled from 'styled-components';
 import Copyright from '../Copyright';
 import Caption from '../Caption';
 import { RequestContext } from '../../contexts/RequestContext';
 
 const LAZYLOAD_OFFSET = 250; // amount of pixels below the viewport to begin loading the image
+const FADE_IN_DURATION = 200; // number of miillisecconds to  apply the fade-in effect
 
-const renderImage = (imageToRender, lazyLoad) =>
+const transitionStyles = {
+  entering: { opacity: 0 },
+  entered: { opacity: 1 },
+  exiting: { opacity: 0 },
+  exited: { opacity: 0 },
+};
+
+export const ImageWrapper = styled.div``;
+
+export const FadeInWrapper = styled.div`
+  transition: ${() => `opacity ${FADE_IN_DURATION}ms ease-in-out`};
+  opacity: ${props => props.opacity || 0};
+`;
+
+export const renderImage = (imageToRender, lazyLoad) =>
   lazyLoad ? (
     <Fragment>
-      <LazyLoad offset={LAZYLOAD_OFFSET} once>
+      <LazyLoad offset={LAZYLOAD_OFFSET} once scroll>
```

Comment:
```suggestion
      <LazyLoad offset={LAZYLOAD_OFFSET} once>
```

We shouldn't need to set this. `scroll` is set to `true` by default.